aioresponses/core.py

```diff
@@ -126,12 +127,18 @@ def _build_response(self, url: 'Union[URL, str]',
             body = json.dumps(payload)
         if not isinstance(body, bytes):
             body = str.encode(body)
+        if request_headers is None:
+            request_headers = {}
         kwargs = {}
         if AIOHTTP_VERSION >= StrictVersion('3.1.0'):
             loop = Mock()
             loop.get_debug = Mock()
             loop.get_debug.return_value = True
-            kwargs['request_info'] = Mock()
+            kwargs['request_info'] = Mock(
```

Comment:
I'm wondering if this could be replace by `aiohttp.RequestInfo`?

Reply:
Done!

Reply:
oops, did it but did not notice you merged! 😬